test/diamorphosis/diamorphosis.test.ts

```diff
@@ -59,4 +59,97 @@ describe('Diamorphosis Test', () => {
       config.honeybadger.environment.should.equal(process.env.NODE_ENV);
     });
   });
+
+  describe('should set json/console loggingvariables', () => {
+    let options;
+
+    beforeEach(() => {
+      options = {
+        diamorphosis: {
+          configFolder: './test/diamorphosis',
+          configPath: path.resolve('./test/diamorphosis/config'),
+          envFolder: path.resolve('./test/diamorphosis/env')
+        }
+      } as OrkaOptions;
+    });
+
+    afterEach(() => {
+      delete require.cache[require.resolve(options.diamorphosis.configPath)];
+    });
+
+    it('nothing set => console:true, json:false, styles:[] ', () => {
```

Comment:
you could use context('when nothing is set config', ()=> it('should be console:true, json:false, styles:[]})

Reply:
Done